Mdule and exports/assignment1-two

```diff
@@ -0,0 +1,17 @@
+exports.frequencyCounter = function (input) {
+  //var input = "asdwqdaabbwwddssafhjkls";
+  var frequency = {}; 
+  for (var index = 0; index < input.length; index++) {
+    var char = input[index];     
+    if (!frequency[char]) {
```

Comment:
@JainVayam Instead of if else, you can also use the ternary operator.
frequency[char] = !frequency[char] ? 1 : frequency[char] + 1;